helix-rest/src/main/java/org/apache/helix/rest/metadatastore/ZkMetadataStoreDirectory.java

```diff
@@ -179,10 +179,10 @@ public void refreshRoutingData(String namespace) {
     }
 
     if (_routingDataMap != null) {
-      MetadataStoreRoutingData newRoutingData =
-          new TrieRoutingData(new TrieRoutingData.TrieNode(null, null, false, null));
+//      MetadataStoreRoutingData newRoutingData =
+//          new TrieRoutingData(new TrieRoutingData.TrieNode(null, null, false, null));
       // TODO call constructRoutingData() here.
-      _routingDataMap.put(namespace, newRoutingData);
+//      _routingDataMap.put(namespace, newRoutingData);
```

Comment:
If the code is not necessary, let's remove it.

Reply:
It's not unnecessary, it just doesn't work anymore after the changes I made in the PR. @narendly Letting you know that this no longer compiles because TrieNode is converted to private as planned. I'm not entirely sure on how to change it so I'm leaving it up to you.

Reply:
The code was just left in there for demonstration purposes. Let's implement this part of the logic to complete ZkMetadataStoreDirectory.

Reply:
@narendly I didn't entirely understand the refresh logic here, actually. That's why I didn't make the change. Let's sync up so I know what to do here.

Reply:
@NealSun96 Let's work together offline :)

Reply:
Resolved by offline discussion.